File: Java/Car.java
public class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer passenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
        
    }

    void prindDataCar() {
        System.out.println(" Licence: " + license + " Driver: " + driver.name + " Passengers: " + passenger);
    }

    public Integer getPassenger(){
        return passenger;
    }

    public void setPassenger(Integer passenger){
        if(passenger == 4){
          this.passenger= passenger;
        }else {
            System.out.println("Necesitas assignar 4 pasajeros");
        }
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }
    
    
}

File: PHP/Car.php
<?php
require_once('Account.php');
class Car
{
    public $id;
    public $license;
    public $driver;
    public $passengers;

    public function __construct($license, $driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar()
    {
        echo "license: $this->license, 
    conductor: {$this->driver->name},
    document: {$this->driver->document},
    Passengers: {$this->passenger}";
    }

    public function getPassenger() {
        return $this->passenger;
    }

    public function setPassenger($passenger) {
        
        if ($passenger == 4) {
            $this->passenger = $passenger;
        }
        else {
            echo "Necesitas asignar 4 pasajeros   ";
        }
    
    }
}
